modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -618,6 +618,12 @@ export class Wallet implements IWallet {
       .send(filteredParams)
       .result();
 
+    response.txInfo.unspents = response.txInfo?.unspents?.map((u) => {
```

Comment:
If `txInfo` or `unspents` are `undefined`, left hand side of the assignment can throw `TypeError`?

Reply:
`unspents[].value` is used within `signTransaction`. 
Is it safe to pass `bigint` for other UTXO coins? Does utxo-lib take care of it?

Reply:
Good observation @saravanan7mani , it would fail if txInfo is undefined which is unlikely to be undefined. Changed it anyway